code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -303,8 +303,7 @@
 			Weaken(stun_pwr*effect_amount)
 
 		if(istype(ears) && (deafen_pwr || damage_pwr))
-			ears.ear_damage += damage_pwr * effect_amount
-			ears.deaf = max(ears.deaf, deafen_pwr * effect_amount)
+			adjustEarDamage(damage_pwr * effect_amount,max(0,(deafen_pwr * effect_amount)-ears.deaf))

```

Comment:
This's kind of hideous to read. Could you put the value calculations and the adjustEarDamage call on separate lines?